### PR TITLE
TR translation folder uploaded

### PR DESCRIPTION
**Pull request by ardacebi**

### CODE REVIEW

File: tr_tr/_messages.js [hunk 1]
```diff
@@ -0,0 +1,1397 @@
...
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Traditional English strings.
+ * @author mckinney@gmail.com (Andrew F. McKinney)
+ */
+'use strict';
+
+goog.provide('AI.Blockly.Msg.en');
```
Comment (barreeeiroo) [inline tr_tr/_messages.js]: You have to set `goog.provide('AI.Blockly.Msg.tr');`

File: tr_tr/_messages.js [hunk 2]
```diff
@@ -0,0 +1,1397 @@
...
+
+goog.require('Blockly.Msg.en');
```
Comment (barreeeiroo) [inline tr_tr/_messages.js]: Here the same `goog.require('AI.Blockly.Msg.tr');`

File: tr_tr/_messages.js [hunk 3]
```diff
@@ -0,0 +1,1397 @@
...
+
+/**
+ * Due to the frequency of long strings, the 80-column wrap rule need not apply
+ * to message files.
+ */
+
+Blockly.Msg.en.switch_language_to_english = {
```
Comment (barreeeiroo) [inline tr_tr/_messages.js]: Here it should be `Blockly.Msg.tr.switch_language_to_turkish = {`

File: tr_tr/_messages.js [hunk 4]
```diff
@@ -0,0 +1,1397 @@
...
+  // Switch language to English.
```
Comment (barreeeiroo) [inline tr_tr/_messages.js]: `// Switch language to Turkish.`

File: tr_tr/_messages.js [hunk 5]
```diff
@@ -0,0 +1,1397 @@
...
+    Blockly.Msg.LANG_PROCEDURES_DEFNORETURN_DEFINE = 'to';
+    Blockly.Msg.LANG_PROCEDURES_DEFNORETURN_PROCEDURE = 'procedure';
+    Blockly.Msg.LANG_PROCEDURES_DEFNORETURN_DO = 'do';
+    Blockly.Msg.LANG_PROCEDURES_DEFNORETURN_COLLAPSED_PREFIX = 'to ';
+    Blockly.Msg.LANG_PROCEDURES_DEFNORETURN_TOOLTIP = 'A procedure that does not return a value.';
+
+    Blockly.Msg.LANG_PROCEDURES_DOTHENRETURN_HELPURL = 'https://docs.makeroid.io/blocks/procedures.html#doreturn';
+    Blockly.Msg.LANG_PROCEDURES_DOTHENRETURN_THEN_RETURN = 'result';
+    Blockly.Msg.LANG_PROCEDURES_DOTHENRETURN_DO = 'do';
+    Blockly.Msg.LANG_PROCEDURES_DOTHENRETURN_RET
```
Comment (barreeeiroo) [inline tr_tr/_messages.js]: `// Initalize language definition to Turkish`

Comment (barreeeiroo) [inline tr_tr/_messages.js]: `Blockly.Msg.tr.switch_blockly_language_to_tr.init();`

Comment (barreeeiroo) [inline tr_tr/_messages.js]: `Blockly.Msg.en.switch_language_to_turkish.init();`

**Review by barreeeiroo — COMMENTED** [review]
I've requested some changes
After they are done I'll merge the PR

File: tr_tr/_messages.js (hunk 1 above)
Comment (ardacebi) [reply tr_tr/_messages.js]: done

File: tr_tr/_messages.js (hunk 2 above)
Comment (ardacebi) [reply tr_tr/_messages.js]: done

File: tr_tr/_messages.js (hunk 5 above)
Comment (ardacebi) [reply tr_tr/_messages.js]: done

Comment (TurboProgramming) [reply tr_tr/_messages.js]: done

Comment (ardacebi) [reply tr_tr/_messages.js]: didn't understand this one.

File: tr_tr/_messages.js (hunk 4 above)
Comment (ardacebi) [reply tr_tr/_messages.js]: done

File: tr_tr/_messages.js (hunk 3 above)
Comment (ardacebi) [reply tr_tr/_messages.js]: done

File: tr_tr/_messages.js (hunk 5 above)
Comment (ardacebi) [reply tr_tr/_messages.js]: didn't understand this one

**Comment by darkyuhoo01** [comment]
barreeeiroo referenced this pull request from commit f06282b about 1 year ago

**Pull request merged**